core/src/test/groovy/noe/server/AS7UndeployTest.groovy

```diff
@@ -0,0 +1,97 @@
+package noe.server
+
+import noe.eap.server.as7.AS7Properties
+import org.junit.AfterClass
+import org.junit.BeforeClass
+import org.junit.Test
+
+import static org.junit.Assert.assertFalse
+import static org.junit.Assert.assertNotNull
+import static org.junit.Assert.assertTrue
+
+class AS7UndeployTest {
+
+    private static final String as7Dir = "test-as7"
+
+    private static File baseDir
+    private static File deploymentsDir
+
+    private static AS7 testServer
+
+    @BeforeClass
+    static void prepare() {
+        baseDir = File.createTempDir('noe', 'ServerAbstract')
+        assertNotNull(baseDir)
+        deploymentsDir = new File(baseDir, "${as7Dir}/standalone/deployments")
+        deploymentsDir.mkdirs()
+        assertNotNull(deploymentsDir)
+
+        //some random value, necessary for AS7 to instantiate (not optimal)
+        System.setProperty("eap.version", "7.0.0")
+        testServer = new AS7(baseDir.getAbsolutePath(), as7Dir)
```

Comment:
@honza-kasik is it worth checking for null here too?

Reply:
Thanks! Doing so in `assertNotNull`.

Reply:
@honza-kasik I can't see the `assertNotNull`

Reply:
https://github.com/web-servers/noe-core/pull/146/files#diff-2d2513fbd8c7990464fe2e7d5c41b5a48e9fd3e6257f60b9ee692abfc909f6a7R24

Reply:
@honza-kasik Sorry, I expressed myself badly, the original question was referred to `testServer` and if it was worth checking it for null, like you have done for the other static class vars.

Reply:
Thanks for the explanation, no worries. I don't think `new` can result to null.